.gitlab-ci.yml

```diff
@@ -49,7 +49,7 @@ build:
 
 test:helm_chart_install:
   rules:
-    - if: '$CI_COMMIT_BRANCH == "master" || $RUN_HELM_CHART_INSTALL == "true"'
+    - if: '$CI_COMMIT_BRANCH == "master" || $CI_COMMIT_BRANCH == "master-next" || $RUN_HELM_CHART_INSTALL == "true"'
```

Comment:
This we can revert

Reply:
I'd keep it as it is, this is handy when we need a feature branch.